### ISSUE TITLE
Non-Chromium compatibility broken in 2.5.2

### ISSUE THREAD

**Issue by Cherry** [gorangajic/isbot#54]
https://github.com/gorangajic/isbot/commit/acb6dcf401f32a48ec7024398f653ea9d7664f53#diff-168726dbe96b3ce427e7fedce31bb0bcR70 throws a `SyntaxError: invalid regexp group` in non-Chromium browsers like Firefox and Edge.

Wrapping in a try/catch doesn't help here, because it's not something that the browser can even parse, thus the `SyntaxError`. 

It looks like you can detect support for this a little better with something like:

```js
try {
	let regex = new RegExp('(?<! cu)bot');
}catch(e){
	// SyntaxError is caught here
}
```

**Comment by Cherry** [comment]
It looks like you can detect support for this a little better with something like:

```js
try {
	let regex = /(?<! cu)bot'/;
}catch(e){
	// error is caught here
}
```

**Comment by Cherry** [comment]
I've opened a PR at https://github.com/gorangajic/isbot/pull/55 to fix this. Let me know if there's anything else you'd like to me change.

**Comment by omrilotan** [comment]
Addressed in version 2.5.3
Thank you
